src/main/java/com/main/projection/PopularFollowerProjection.java

```diff
@@ -0,0 +1,10 @@
+package com.main.projection;
+
+public interface PopularFollowerProjection {
```

Comment:
This should be a DTO

Reply:
a projection is a part entity, I don't think is a good practice with a direct relation with the persistence layer